packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -94,7 +94,7 @@ export const ProposalPreview = () => {
   const myVote = proposal?.votes.find((vote) => vote.voter.id === active?.id && vote.votingRound === currentVotingRound)
   const myVoteStatus = myVote?.voteKind
 
-  if (!proposal || !votes) {
+  if (!proposal || !votes || !opening) {
     if (!proposal && !isLoading) {
       history.replace('/404')
```

Comment:
This may be out of scope but i wonder if we want a 404 page instead of routing back to `/proposals` for example?

Reply:
thanks for finding this!

Reply:
I believe we should drop this 404 altogether. Follow-up: #4108